okhttp-testing-support/src/main/kotlin/okhttp3/RecordingEventListener.kt

```diff
@@ -55,7 +55,9 @@ import org.assertj.core.api.Assertions.assertThat
 import org.assertj.core.data.Offset
 import org.junit.jupiter.api.Assertions.assertTrue
 
-open class RecordingEventListener : EventListener() {
+open class RecordingEventListener(
+  private val enforceOrder: Boolean = true
```

Comment:
Maybe this needs some doc about the order it's gonna enforce and maybe when it's useful or not